### PR TITLE
Feat: add background:none to the :focus-visible selector

### PR DESCRIPTION
**Pull request by gjanblaszczyk**
## Description
A proposed fix for videojs/video.js#5557
Please watch below video to see how the captions menu will work after proposed changes.
videojsmenuclickafterfix

## Requirements Checklist
- [X] Feature implemented / Bug fixed
- [X] If necessary, more likely in a feature request than a bug fix
  - [X] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](http://jsbin.com/axedog/edit?html,output))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

**Comment by gkatsev** [comment]
Yeah, I'm not sure what's happening with tests. I updated deps this week which seems to help but I'm not sure.

File: src/css/components/menu/_menu.scss [hunk 1]
```diff
@@ -43,13 +43,13 @@
 }
 
 .vjs-menu li.vjs-menu-item:focus,
-.vjs-menu li.vjs-menu-item:hover {
+.vjs-menu .vjs-menu-content li.vjs-menu-item:hover {
```
Comment (gkatsev) [inline src/css/components/menu/_menu.scss]: why increase the specificity?

Comment (gjanblaszczyk) [reply src/css/components/menu/_menu.scss]: The CSS ".js-focus-visible .video-js *:focus:not(.focus-visible)" selector has higher prioryty than ".vjs-menu li.vjs-menu-item:hover". So if the first menu item gets focus, menu item doesn't change background color on hover. That's way I increased the specificity to the hover CSS selector to resolve above problem.

Comment (gkatsev) [reply src/css/components/menu/_menu.scss]: Ah, hm... I'd be worried that increasing this specificity would break some custom skins

Comment (gjanblaszczyk) [reply src/css/components/menu/_menu.scss]: @gkatsev Does it look better now?

**Comment by gkatsev** [comment]
@gjanblaszczyk can you rebase/merge in master? We have a fix for the test failures.

**Comment by gjanblaszczyk** [comment]
@gkatsev Ok done and now all tests have passed.

**Pull request merged**